samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', filepath = 'path/to/image.png') {
+  // [START dlp_inspect_image_file]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
```

Comment:
Instead of passing in the `projectId`, or asking the user to specify it - can we just say:
```js
const projectId = await dlp.auth.getProjectId();
```

This would cut down on *a lot* of boilerplate.  And in reality - we don't ever really want users hard coding this, right?

Reply:
See https://github.com/googleapis/nodejs-dlp/pull/154#discussion_r262211255

Reply:
Is `getProjectId()` something implemented across the 7 langs? Is it anything more than an environment variable check? (and, if so, which one(s)?) I remember trying to come up with consensus around this feature years ago, but am not aware of where/when/how it landed.

While it's true that we would never want users to hard-core their project ID, I find the "Your Google Cloud Project ID" variables useful to be explicit about how to use the sample with your given GCP Project.